### PR TITLE
fix document regarding accessiblity_id_find

### PR DESCRIPTION
**Pull request by KazuCocoa**
fix https://github.com/appium/ruby_lib/issues/276

### CODE REVIEW

File: lib/appium_lib/device/device.rb [hunk 1]
```diff
@@ -336,7 +336,7 @@ def create_bridge_command(method, verb, path)
         end
       end
 
-      # @!method accessiblity_id_find
+      # @!method find_element/s
```
Comment (bootstraponline) [inline lib/appium_lib/device/device.rb]: This is still wrong. `find_element/s` is not a method.

I think we want:

> @!method find_element
> @!method find_elements

**Comment by KazuCocoa** [comment]
i fixed.
Should I also commit `docs/android_docs.md` and `docs/ios_docs.md` generated with `$ ruby docs_gen/make_docs.rb` ?

**Comment by bootstraponline** [comment]
> Should I also commit docs/android_docs.md and docs/ios_docs.md generated with $ ruby docs_gen/make_docs.rb ?

Sure, it's better to just do `rake docs`

**Comment by KazuCocoa** [comment]
My branch was old one. So, i send pr again. And close this.

Fix docs of find elements #372

**Pull request closed without merging**